docs/advanced/addons/managed-dhcp.md

```diff
@@ -0,0 +1,248 @@
+---
+sidebar_position: 6
+sidebar_label: Managed DHCP
+title: "Managed DHCP"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/advanced/managed-dhcp"/>
+</head>
+
+_Available as of v1.3.0_
+
+Beginning with v1.3.0, users can configure IP pool information and serve IP addresses to virtual machines running on Harvester clusters via DHCP.
```

Comment:
Beginning with v1.3.0, users can configure IP pool information and serve IP addresses to virtual machines running on Harvester clusters via DHCP.
->
Beginning with v1.3.0, users can configure IP pool information and serve IP addresses to virtual machines running on Harvester clusters via the embedded `Managed DHCP` feature. It is an alternative of the standalone `DHCP server` to simplify the cluster deployment.

:::note
Users need to ensure the network connectivity and plan the IP pools in advance. Harvester is using the planned infrastructure network, not managing it.
:::